modules/nlu/src/backend/language/language-provider.ts

```diff
@@ -150,13 +159,19 @@ export class RemoteLanguageProvider implements LanguageProvider {
     return this
   }
 
-  private extractLangServerVersion(data) {
+  private extractLangServerInfo(data) {
     const version = semver.valid(semver.coerce(data.version))
 
     if (!version) {
       throw new Error('Lang server has an invalid version')
     }
-    this._state.langServerVersion = semver.clean(version)
+    const langServerInfo: LangServerInfo = {
```

Comment:
just add a comment that state this will set lang server info on each iterations in case multiple lang providers are defined and that only the last info will be taken into account (hopefully the user will have configured its different lang servers with same dims and same domain ..

We should also add a todo comment (validate all lang providers share the same configs)